### ISSUE TITLE
ALISA goal - requirements check

### ISSUE THREAD

**Issue by dpgluch** [osate/osate2#2630]
**Summary**
There is an error message that a goal does not have a corresponding requirement when there are corresponding requirements, i.e. requirements that reference the goal. See attached screen shots.

**Expected behavior**
No error message.

**Actual behavior**
An error message is presented

**Steps To Reproduce**

- If you can provide a small model or test case that demonstrates the issue, it will be much easier to debug.
- Paste small code snippets in the three ```backticks``` below, larger ones should be put in a [gist](gist.github.com)

```aadl
Paste your model here
```

**Screenshots**
![goals error1](https://user-images.githubusercontent.com/24554086/113326983-61741e80-92e8-11eb-9880-3639031721ea.PNG)
![goals error2](https://user-images.githubusercontent.com/24554086/113326994-646f0f00-92e8-11eb-9dda-f2f138e3bc80.PNG)

**Desktop (please complete the following information):**
 - OSATE Version:  OSATE2	2.9.2.v20210324-1240	osate2	null
 - Operating System: Windows 10

**Additional context**

**Comment by lwrage** [comment]
Could you include a minimal model that shows the issue, please?

**Comment by dpgluch** [comment]
Lutz,

I added an example.
In putting both the system model and ALISA files in one project, found that the error does not occur.  More info on github.

Dave

**Comment by dpgluch** [comment]
I have attached an example project.  There is no error when the system model is in the same project as the ALISA files.  In the attached example, if the package ErrorExample_Pkg is contained in another project, the error message occurs, even though the project containing the system model is referenced .  Found this out in putting together the example.

[autonomousAircraft.zip](https://github.com/osate/osate2/files/6258968/autonomousAircraft.zip)
